### ISSUE TITLE
Can't disable pip3

### ISSUE THREAD

**Issue by yochananmarqos** [r-darwish/topgrade#689]
**What did you expect to happen?**
`pip3` should be disabled:

`~/.config/topgrade.toml`:
```
disable = ["cargo", "node", "pip3"]
```

**What actually happened?**
`pip` 20.3.1 was installed to `~/local/` unnecessarily. I already have `python-pip` 20.3.1-1 installed.

```
❯ which pip
/usr/bin/pip
```
```
❯ pip --version
pip 20.3.1 from /usr/lib/python3.9/site-packages/pip (python 3.9)
```

**Additional details:**
- Which operating system or Linux distribution are you using?
Manjaro GNOME

- How did you install Topgrade?
Manjaro community package which is the same as the AUR package

`topgrade -v`:
<details><pre>
 2021-04-11T21:37:10.300Z DEBUG topgrade::config > Configuration at /home/yochanan/.config/topgrade.toml
 2021-04-11T21:37:10.303Z DEBUG topgrade::config > Loaded configuration: ConfigFile { pre_commands: Some({}), post_commands: None, commands: Some({}), git_repos: None, predefined_git_repos: None, disable: Some([Cargo, Node, Pip3]), ignore_failures: None, remote_topgrades: None, remote_topgrade_path: None, ssh_arguments: None, git_arguments: None, tmux_arguments: None, set_title: None, assume_yes: None, yay_arguments: None, no_retry: None, run_in_tmux: None, cleanup: None, notify_each_step: None, accept_all_windows_updates: None, bashit_branch: None, only: None, composer: Some(Composer { self_update: None }), brew: Some(Brew { greedy_cask: None }), linux: Some(Linux { yay_arguments: None, trizen_arguments: None, dnf_arguments: None, apt_arguments: None, enable_tlmgr: None, redhat_distro_sync: None, emerge_sync_flags: None, emerge_update_flags: None }), git: Some(Git { max_concurrency: None, arguments: None, repos: None, pull_predefined: None }), windows: Some(Windows { accept_all_updates: None, self_rename: None, open_remotes_in_new_terminal: None }), vagrant: None }
 2021-04-11T21:37:10.303Z DEBUG topgrade::utils  > Detected "/usr/bin/notify-send" as "notify-send"
 2021-04-11T21:37:10.303Z DEBUG topgrade         > Version: 6.8.0
 2021-04-11T21:37:10.303Z DEBUG topgrade         > OS: x86_64-unknown-linux-gnu
 2021-04-11T21:37:10.303Z DEBUG topgrade         > Args { inner: ["topgrade", "-v"] }
 2021-04-11T21:37:10.303Z DEBUG topgrade         > Binary path: Ok("/usr/bin/topgrade")
 2021-04-11T21:37:10.303Z DEBUG topgrade         > Self Update: false
 2021-04-11T21:37:10.303Z DEBUG topgrade::utils  > Detected "/usr/bin/git" as "git"
 2021-04-11T21:37:10.303Z DEBUG topgrade::utils  > Cannot find "doas"
 2021-04-11T21:37:10.303Z DEBUG topgrade::utils  > Detected "/usr/bin/sudo" as "sudo"
 2021-04-11T21:37:10.303Z DEBUG topgrade::utils  > Cannot find "pwsh"
 2021-04-11T21:37:10.303Z DEBUG topgrade::utils  > Cannot find "powershell"
 2021-04-11T21:37:10.303Z DEBUG topgrade::runner > Step "System update"

―― 15:37:10 - System update ――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-04-11T21:37:10.303Z DEBUG topgrade::utils  > Cannot find "powerpill"
 2021-04-11T21:37:10.303Z DEBUG topgrade::steps::os::linux > Running Arch update with path: "/usr/bin:/home/yochanan/bin:/home/yochanan/.local/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl"
 2021-04-11T21:37:10.303Z DEBUG topgrade::utils            > Detected "/usr/bin/yay" as "yay"
 2021-04-11T21:37:10.303Z DEBUG topgrade::executor         > Running "/usr/bin/yay" "-Pw"
 2021-04-11T21:37:11.308Z DEBUG topgrade::executor         > Running "/usr/bin/yay" "--pacman" "/usr/bin/pacman" "-Syu" "--devel"
:: Synchronizing package databases...
 core is up to date
 extra is up to date
 community is up to date
 multilib is up to date
:: Starting full system upgrade...
warning: gjs: ignoring package upgrade (2:1.66.2-1 => 2:1.68.0-2)
warning: lutris: local (0.5.8.3-2.1) is newer than community (0.5.8.3-2)
warning: pamac-cli: local (11.0.0beta.r0.g93fefcd-1) is newer than extra (10.1.0beta2-1)
warning: pamac-common: local (11.0.0beta.r0.g93fefcd-1) is newer than extra (10.1.0beta2-1)
warning: pamac-gnome-integration: local (11.0.0beta.r0.g93fefcd-1) is newer than extra (10.1.0beta2-1)
warning: pamac-gtk: local (11.0.0beta.r0.g93fefcd-1) is newer than extra (10.1.0beta2-1)
warning: python-dropbox: local (11.6.0-1) is newer than community (10.10.0-2)
warning: python-markups: local (3.1.1-0) is newer than community (3.0.0-10)
warning: retext: local (7.2.1-1.1) is newer than community (7.2.1-1)
warning: timeshift: local (20.11.1+3+g08d0e59-3) is newer than community (20.11.1.r3.g08d0e59-2)
 there is nothing to do
:: Searching databases for updates...
:: Searching AUR for updates...
 -> gjs: ignoring package upgrade (2:1.66.2-1 => 2:1.68.0-2)
 -> lutris: local (0.5.8.3-2.1) is newer than community (0.5.8.3-2)
 -> pamac-cli: local (11.0.0beta.r0.g93fefcd-1) is newer than extra (10.1.0beta2-1)
 -> pamac-common: local (11.0.0beta.r0.g93fefcd-1) is newer than extra (10.1.0beta2-1)
 -> pamac-gnome-integration: local (11.0.0beta.r0.g93fefcd-1) is newer than extra (10.1.0beta2-1)
 -> pamac-gtk: local (11.0.0beta.r0.g93fefcd-1) is newer than extra (10.1.0beta2-1)
 -> python-dropbox: local (11.6.0-1) is newer than community (10.10.0-2)
 -> python-markups: local (3.1.1-0) is newer than community (3.0.0-10)
 -> retext: local (7.2.1-1.1) is newer than community (7.2.1-1)
 -> timeshift: local (20.11.1+3+g08d0e59-3) is newer than community (20.11.1.r3.g08d0e59-2)
:: Checking development packages...
 -> libadwaita-git: ignoring package upgrade (1.1.0.r1596.g65ac525-1 => 1:r1560.4f6e828-2)
 -> mp3tag: ignoring package upgrade (3.06a-1 => 3.06-1)
 -> nautilus-copy-path: ignoring package upgrade (1.5.2-1 => 1.5.2-2)
 -> python-sentry_sdk: ignoring package upgrade (0.19.5-1 => 1.0.0-1)
 -> chromecast_mpris: local (0.7.5-1) is newer than AUR (0.7.1-1)
 -> lebiniou: local (3.55.0-1.1) is newer than AUR (3.55.0-1)
 -> python-magic-ahupp: local (0.4.22-1) is newer than AUR (0.4.20-2)
 -> python-pychromecast: local (9.1.2-1) is newer than AUR (9.1.1-1)
 -> scrcpy: local (1.17-1.1) is newer than AUR (1.17-1)
 -> vulkan-caps-viewer: local (3.0-2) is newer than AUR (3.0-1)
 -> Missing AUR Packages:  arch-wiki-docs  commit  g-assist  gnome-getting-started-docs  keyboard-color-switcher-git  python-stone  s76-keyboard-colors  x205ta-create-iso
 -> Orphaned AUR Packages:  libmagick6
 there is nothing to do
 2021-04-11T21:37:17.815Z DEBUG topgrade::runner           > Step "etc-update"
 2021-04-11T21:37:17.815Z DEBUG topgrade::runner           > Step "Brew"
 2021-04-11T21:37:17.815Z DEBUG topgrade::runner           > Step "yadm"
 2021-04-11T21:37:17.815Z DEBUG topgrade::runner           > Step "nix"
 2021-04-11T21:37:17.815Z DEBUG topgrade::runner           > Step "home-manager"
 2021-04-11T21:37:17.815Z DEBUG topgrade::runner           > Step "asdf"
 2021-04-11T21:37:17.815Z DEBUG topgrade::utils            > Path "/home/yochanan/.config/emacs" doesn't exist
 2021-04-11T21:37:17.815Z DEBUG topgrade::utils            > Path "/home/yochanan/.emacs.d" doesn't exist
 2021-04-11T21:37:17.815Z DEBUG topgrade::steps::git       > /home/yochanan/.doom.d does not exists
 2021-04-11T21:37:17.815Z DEBUG topgrade::steps::git       > /home/yochanan/.vim does not exists
 2021-04-11T21:37:17.815Z DEBUG topgrade::steps::git       > Checking if /home/yochanan/.config/nvim is a git repository
 2021-04-11T21:37:17.816Z TRACE topgrade::executor         > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-04-11T21:37:17.816Z DEBUG topgrade::steps::git       > /home/yochanan/.zshrc is a file. Checking /home/yochanan
 2021-04-11T21:37:17.816Z DEBUG topgrade::steps::git       > Checking if /home/yochanan is a git repository
 2021-04-11T21:37:17.817Z TRACE topgrade::executor         > Output of "/usr/bin/git" "rev-parse" "--show-toplevel": Output { status: ExitStatus(ExitStatus(32768)), stdout: "", stderr: "fatal: not a git repository (or any of the parent directories): .git\n" }
 2021-04-11T21:37:17.817Z DEBUG topgrade::steps::git       > /home/yochanan/.tmux does not exists
 2021-04-11T21:37:17.817Z DEBUG topgrade::steps::git       > /home/yochanan/.config/fish does not exists
 2021-04-11T21:37:17.817Z DEBUG topgrade::steps::git       > /home/yochanan/.config/openbox does not exists
 2021-04-11T21:37:17.817Z DEBUG topgrade::steps::git       > /home/yochanan/.config/bspwm does not exists
 2021-04-11T21:37:17.817Z DEBUG topgrade::steps::git       > /home/yochanan/.config/i3 does not exists
 2021-04-11T21:37:17.817Z DEBUG topgrade::steps::git       > /home/yochanan/.config/sway does not exists
 2021-04-11T21:37:17.817Z DEBUG topgrade::runner           > Step "Git repositories"
 2021-04-11T21:37:17.817Z DEBUG topgrade::runner           > Step "zr"
 2021-04-11T21:37:17.817Z DEBUG topgrade::utils            > Detected "/usr/bin/zsh" as "zsh"
 2021-04-11T21:37:17.817Z DEBUG topgrade::runner           > Step "antibody"
 2021-04-11T21:37:17.817Z DEBUG topgrade::utils            > Detected "/usr/bin/zsh" as "zsh"
 2021-04-11T21:37:17.817Z DEBUG topgrade::runner           > Step "antigen"
 2021-04-11T21:37:17.817Z DEBUG topgrade::utils            > Detected "/usr/bin/zsh" as "zsh"
 2021-04-11T21:37:17.817Z DEBUG topgrade::utils            > Path "/home/yochanan/.zshrc" exists
 2021-04-11T21:37:17.817Z DEBUG topgrade::runner           > Step "zplug"
 2021-04-11T21:37:17.817Z DEBUG topgrade::utils            > Detected "/usr/bin/zsh" as "zsh"
 2021-04-11T21:37:17.817Z DEBUG topgrade::utils            > Path "/home/yochanan/.zshrc" exists
 2021-04-11T21:37:17.817Z DEBUG topgrade::runner           > Step "zinit"
 2021-04-11T21:37:17.817Z DEBUG topgrade::utils            > Detected "/usr/bin/zsh" as "zsh"
 2021-04-11T21:37:17.817Z DEBUG topgrade::utils            > Path "/home/yochanan/.zshrc" exists
 2021-04-11T21:37:17.817Z DEBUG topgrade::runner           > Step "zim"
 2021-04-11T21:37:17.817Z DEBUG topgrade::utils            > Detected "/usr/bin/zsh" as "zsh"
 2021-04-11T21:37:17.988Z TRACE topgrade::executor         > Output of "zsh" "-c" "[[ -n ${ZIM_HOME} ]] && print -n ${ZIM_HOME}": Output { status: ExitStatus(ExitStatus(256)), stdout: "\u{1b}[?25l\u{1b}[?7l\u{1b}[0m\u{1b}[32m\u{1b}[1m██████████████████  ████████\n██████████████████  ████████\n██████████████████  ████████\n██████████████████  ████████\n████████            ████████\n████████  ████████  ████████\n████████  ████████  ████████\n████████  ████████  ████████\n████████  ████████  ████████\n████████  ████████  ████████\n████████  ████████  ████████\n████████  ████████  ████████\n████████  ████████  ████████\n████████  ████████  ████████\u{1b}[0m\n\u{1b}[14A\u{1b}[9999999D\u{1b}[31C\u{1b}[0m\u{1b}[1m\u{1b}[32m\u{1b}[1myochanan\u{1b}[0m@\u{1b}[32m\u{1b}[1mmanjaro\u{1b}[0m \n\u{1b}[31C\u{1b}[0m----------------\u{1b}[0m \n\u{1b}[31C\u{1b}[0m\u{1b}[32m\u{1b}[1mOS\u{1b}[0m\u{1b}[0m:\u{1b}[0m Manjaro Linux x86_64\u{1b}[0m \n\u{1b}[31C\u{1b}[0m\u{1b}[32m\u{1b}[1mHost\u{1b}[0m\u{1b}[0m:\u{1b}[0m Serval WS serw12\u{1b}[0m \n\u{1b}[31C\u{1b}[0m\u{1b}[32m\u{1b}[1mKernel\u{1b}[0m\u{1b}[0m:\u{1b}[0m 5.11.13-1-MANJARO\u{1b}[0m \n\u{1b}[31C\u{1b}[0m\u{1b}[32m\u{1b}[1mUptime\u{1b}[0m\u{1b}[0m:\u{1b}[0m 1 day, 1 hour, 2 mins\u{1b}[0m \n\u{1b}[31C\u{1b}[0m\u{1b}[32m\u{1b}[1mPackages\u{1b}[0m\u{1b}[0m:\u{1b}[0m 1958 (pacman)\u{1b}[0m \n\u{1b}[31C\u{1b}[0m\u{1b}[32m\u{1b}[1mShell\u{1b}[0m\u{1b}[0m:\u{1b}[0m zsh 5.8\u{1b}[0m \n\u{1b}[31C\u{1b}[0m\u{1b}[32m\u{1b}[1mResolution\u{1b}[0m\u{1b}[0m:\u{1b}[0m 1920x1080, 1920x1080\u{1b}[0m \n\u{1b}[31C\u{1b}[0m\u{1b}[32m\u{1b}[1mDE\u{1b}[0m\u{1b}[0m:\u{1b}[0m GNOME 40.0\u{1b}[0m \n\u{1b}[31C\u{1b}[0m\u{1b}[32m\u{1b}[1mWM\u{1b}[0m\u{1b}[0m:\u{1b}[0m Mutter\u{1b}[0m \n\u{1b}[31C\u{1b}[0m\u{1b}[32m\u{1b}[1mWM Theme\u{1b}[0m\u{1b}[0m:\u{1b}[0m Adwaita\u{1b}[0m \n\u{1b}[31C\u{1b}[0m\u{1b}[32m\u{1b}[1mTheme\u{1b}[0m\u{1b}[0m:\u{1b}[0m Adwaita-dark [GTK2/3]\u{1b}[0m \n\u{1b}[31C\u{1b}[0m\u{1b}[32m\u{1b}[1mIcons\u{1b}[0m\u{1b}[0m:\u{1b}[0m Papirus-Dark [GTK2/3]\u{1b}[0m \n\u{1b}[31C\u{1b}[0m\u{1b}[32m\u{1b}[1mTerminal\u{1b}[0m\u{1b}[0m:\u{1b}[0m topgrade\u{1b}[0m \n\u{1b}[31C\u{1b}[0m\u{1b}[32m\u{1b}[1mCPU\u{1b}[0m\u{1b}[0m:\u{1b}[0m AMD Ryzen 7 3700X (16) @ 3.600GHz\u{1b}[0m \n\u{1b}[31C\u{1b}[0m\u{1b}[32m\u{1b}[1mGPU\u{1b}[0m\u{1b}[0m:\u{1b}[0m NVIDIA GeForce GTX 1660 Ti Mobile\u{1b}[0m \n\u{1b}[31C\u{1b}[0m\u{1b}[32m\u{1b}[1mMemory\u{1b}[0m\u{1b}[0m:\u{1b}[0m 8703MiB / 32067MiB\u{1b}[0m \n\n\u{1b}[31C\u{1b}[30m\u{1b}[40m   \u{1b}[31m\u{1b}[41m   \u{1b}[32m\u{1b}[42m   \u{1b}[33m\u{1b}[43m   \u{1b}[34m\u{1b}[44m   \u{1b}[35m\u{1b}[45m   \u{1b}[36m\u{1b}[46m   \u{1b}[37m\u{1b}[47m   \u{1b}[m\n\u{1b}[31C\u{1b}[38;5;8m\u{1b}[48;5;8m   \u{1b}[38;5;9m\u{1b}[48;5;9m   \u{1b}[38;5;10m\u{1b}[48;5;10m   \u{1b}[38;5;11m\u{1b}[48;5;11m   \u{1b}[38;5;12m\u{1b}[48;5;12m   \u{1b}[38;5;13m\u{1b}[48;5;13m   \u{1b}[38;5;14m\u{1b}[48;5;14m   \u{1b}[38;5;15m\u{1b}[48;5;15m   \u{1b}[m\n\n\n\u{1b}[?25h\u{1b}[?7h", stderr: "" }
 2021-04-11T21:37:17.988Z DEBUG topgrade::runner           > Step "oh-my-zsh"
 2021-04-11T21:37:17.988Z DEBUG topgrade::utils            > Detected "/usr/bin/zsh" as "zsh"
 2021-04-11T21:37:17.988Z DEBUG topgrade::runner           > Step "fisher"
 2021-04-11T21:37:17.988Z DEBUG topgrade::runner           > Step "bash-it"
 2021-04-11T21:37:17.988Z DEBUG topgrade::runner           > Step "oh-my-fish"
 2021-04-11T21:37:17.988Z DEBUG topgrade::runner           > Step "tmux"
 2021-04-11T21:37:17.988Z DEBUG topgrade::runner           > Step "TLDR"
 2021-04-11T21:37:17.988Z DEBUG topgrade::utils            > Detected "/usr/bin/tldr" as "tldr"

―― 15:37:17 - TLDR ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-04-11T21:37:17.988Z DEBUG topgrade::executor         > Running "/usr/bin/tldr" "--update"
No changes at tldr-pages repository.
 2021-04-11T21:37:18.567Z DEBUG topgrade::runner           > Step "pearl"
 2021-04-11T21:37:18.567Z DEBUG topgrade::runner           > Step "SDKMAN!"
 2021-04-11T21:37:18.567Z DEBUG topgrade::utils            > Detected "/usr/bin/bash" as "bash"
 2021-04-11T21:37:18.567Z DEBUG topgrade::runner           > Step "apm"
 2021-04-11T21:37:18.567Z DEBUG topgrade::runner           > Step "vscode"
 2021-04-11T21:37:18.567Z DEBUG topgrade::runner           > Step "vscodium"
 2021-04-11T21:37:18.567Z DEBUG topgrade::runner           > Step "fossil"
 2021-04-11T21:37:18.567Z DEBUG topgrade::runner           > Step "rustup"
 2021-04-11T21:37:18.567Z DEBUG topgrade::runner           > Step ".NET"
 2021-04-11T21:37:18.567Z DEBUG topgrade::runner           > Step "choosenim"
 2021-04-11T21:37:18.567Z DEBUG topgrade::runner           > Step "Flutter"
 2021-04-11T21:37:18.567Z DEBUG topgrade::runner           > Step "Emacs"
 2021-04-11T21:37:18.567Z DEBUG topgrade::runner           > Step "opam"
 2021-04-11T21:37:18.567Z DEBUG topgrade::runner           > Step "vcpkg"
 2021-04-11T21:37:18.567Z DEBUG topgrade::runner           > Step "pipx"
 2021-04-11T21:37:18.567Z DEBUG topgrade::runner           > Step "pip3"
 2021-04-11T21:37:18.567Z DEBUG topgrade::utils            > Detected "/usr/bin/pip3" as "pip3"

―― 15:37:18 - pip3 ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
 2021-04-11T21:37:18.568Z DEBUG topgrade::executor         > Running "/usr/bin/pip3" "install" "--upgrade" "--user" "pip"
Requirement already satisfied: pip in /usr/lib/python3.9/site-packages (20.3.1)
Collecting pip
  Using cached pip-21.0.1-py3-none-any.whl (1.5 MB)
Installing collected packages: pip
Successfully installed pip-21.0.1
 2021-04-11T21:37:20.484Z DEBUG topgrade::runner           > Step "stack"
 2021-04-11T21:37:20.485Z DEBUG topgrade::runner           > Step "tlmgr"
 2021-04-11T21:37:20.485Z DEBUG topgrade::runner           > Step "myrepos"
 2021-04-11T21:37:20.485Z DEBUG topgrade::runner           > Step "jetpack"
 2021-04-11T21:37:20.485Z DEBUG topgrade::runner           > Step "vim"
 2021-04-11T21:37:20.485Z DEBUG topgrade::runner           > Step "Neovim"
 2021-04-11T21:37:20.485Z DEBUG topgrade::runner           > Step "voom"
 2021-04-11T21:37:20.485Z DEBUG topgrade::runner           > Step "deno"
 2021-04-11T21:37:20.485Z DEBUG topgrade::runner           > Step "composer"
 2021-04-11T21:37:20.485Z DEBUG topgrade::runner           > Step "krew"
 2021-04-11T21:37:20.485Z DEBUG topgrade::runner           > Step "gem"
 2021-04-11T21:37:20.485Z DEBUG topgrade::runner           > Step "sheldon"
 2021-04-11T21:37:20.485Z DEBUG topgrade::runner           > Step "rtcl"
 2021-04-11T21:37:20.485Z DEBUG topgrade::runner           > Step "gcloud"
 2021-04-11T21:37:20.485Z DEBUG topgrade::runner           > Step "micro"
 2021-04-11T21:37:20.485Z DEBUG topgrade::utils            > Detected "/usr/bin/micro" as "micro"

―― 15:37:20 - micro ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
Checking for plugin updates
Nothing to install / update
 2021-04-11T21:37:20.988Z DEBUG topgrade::runner           > Step "raco"
 2021-04-11T21:37:20.988Z DEBUG topgrade::runner           > Step "Flatpak"
 2021-04-11T21:37:20.988Z DEBUG topgrade::utils            > Detected "/usr/bin/flatpak" as "flatpak"

―― 15:37:20 - Flatpak User Packages ――――――――――――――――――――――――――――――――――――――――――――
 2021-04-11T21:37:20.988Z DEBUG topgrade::executor         > Running "/usr/bin/flatpak" "update" "--user" "-y"
Looking for updates…
Nothing to do.
 2021-04-11T21:37:21.006Z DEBUG topgrade::executor         > Running "/usr/bin/flatpak" "update" "--system" "-y"
Looking for updates…
Nothing to do.
 2021-04-11T21:37:21.023Z DEBUG topgrade::runner           > Step "snap"
 2021-04-11T21:37:21.023Z DEBUG topgrade::runner           > Step "pihole"
 2021-04-11T21:37:21.023Z DEBUG topgrade::runner           > Step "Firmware upgrades"
 2021-04-11T21:37:21.023Z DEBUG topgrade::utils            > Detected "/usr/bin/fwupdmgr" as "fwupdmgr"
 2021-04-11T21:37:21.024Z TRACE topgrade::executor         > Output of "uname" "-r": Output { status: ExitStatus(ExitStatus(0)), stdout: "5.11.13-1-MANJARO\n", stderr: "" }
 2021-04-11T21:37:21.024Z DEBUG topgrade::steps::os::linux > Uname output: 5.11.13-1-MANJARO

―― 15:37:21 - Firmware upgrades ――――――――――――――――――――――――――――――――――――――――――――――――
 2021-04-11T21:37:21.024Z DEBUG topgrade::executor         > Running "/usr/bin/fwupdmgr" "refresh"
Updating lvfs
Downloading…             [***************************************]
Successfully downloaded new metadata: 0 local devices supported
 2021-04-11T21:37:21.154Z DEBUG topgrade::executor         > Running "/usr/bin/fwupdmgr" "upgrade"
Devices with no available firmware updates: 
 • Samsung SSD 970 EVO Plus 1TB
 • System Firmware
 • UEFI Device Firmware
 • WDC WD10SPZX-75Z10T2
 • WDC WDS100T2B0C-00PXH0
No updatable devices
 2021-04-11T21:37:21.173Z DEBUG topgrade::runner           > Step "Restarts"
 2021-04-11T21:37:21.173Z DEBUG topgrade::utils            > Detected "/usr/bin/needrestart" as "needrestart"

―― 15:37:21 - Check for needed restarts ――――――――――――――――――――――――――――――――――――――――
 2021-04-11T21:37:21.174Z DEBUG topgrade::executor         > Running "/usr/bin/sudo" "/usr/bin/needrestart"
Scanning processes...                                                                         
Scanning candidates...                                                                        
Scanning processor microcode...                                                               
Scanning linux images...                                                                      

Running kernel seems to be up-to-date.

The processor microcode seems to be up-to-date.

Restarting services...
Services to be restarted:
Restart «user@1000.service»? [yNas?] n
Service restarts being deferred:
 systemctl restart user@1000.service

No containers need to be restarted.

No user sessions are running outdated binaries.

―― 15:37:24 - Summary ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
System update: OK
etc-update: SKIPPED: Cannot find "etc-update" in PATH
Brew: SKIPPED: Cannot find "brew" in PATH
yadm: SKIPPED: Cannot find "yadm" in PATH
nix: SKIPPED: Cannot find "nix" in PATH
home-manager: SKIPPED: Cannot find "home-manager" in PATH
asdf: SKIPPED: Cannot find "asdf" in PATH
Git repositories: SKIPPED: No repositories to pull
zr: SKIPPED: Cannot find "zr" in PATH
antibody: SKIPPED: Cannot find "antibody" in PATH
antigen: SKIPPED: Path "/home/yochanan/antigen.zsh" doesn't exist
zplug: SKIPPED: Path "/home/yochanan/.zplug" doesn't exist
zinit: SKIPPED: Path "/home/yochanan/.zinit" doesn't exist
zim: SKIPPED: Path "/home/yochanan/.zim" doesn't exist
oh-my-zsh: SKIPPED: Path "/home/yochanan/.oh-my-zsh" doesn't exist
fisher: SKIPPED: Cannot find "fish" in PATH
bash-it: SKIPPED: Path "/home/yochanan/.bash_it" doesn't exist
oh-my-fish: SKIPPED: Cannot find "fish" in PATH
tmux: SKIPPED: Path "/home/yochanan/.tmux/plugins/tpm/bin/update_plugins" doesn't exist
TLDR: OK
pearl: SKIPPED: Cannot find "pearl" in PATH
SDKMAN!: SKIPPED: Path "/home/yochanan/.sdkman/bin/sdkman-init.sh" doesn't exist
apm: SKIPPED: Cannot find "apm" in PATH
vscode: SKIPPED: Cannot find "code" in PATH
vscodium: SKIPPED: Cannot find "codium" in PATH
fossil: SKIPPED: Cannot find "fossil" in PATH
rustup: SKIPPED: Cannot find "rustup" in PATH
.NET: SKIPPED: Cannot find "dotnet" in PATH
choosenim: SKIPPED: Cannot find "choosenim" in PATH
Flutter: SKIPPED: Cannot find "flutter" in PATH
Emacs: SKIPPED: Cannot find "emacs" in PATH
opam: SKIPPED: Cannot find "opam" in PATH
vcpkg: SKIPPED: Cannot find "vcpkg" in PATH
pipx: SKIPPED: Cannot find "pipx" in PATH
pip3: OK
stack: SKIPPED: Cannot find "stack" in PATH
tlmgr: SKIPPED: tlmgr must be explicity enabled in the configuration to run in Linux
myrepos: SKIPPED: Cannot find "mr" in PATH
jetpack: SKIPPED: Cannot find "jetpack" in PATH
vim: SKIPPED: Cannot find "vim" in PATH
Neovim: SKIPPED: Cannot find "nvim" in PATH
voom: SKIPPED: Cannot find "voom" in PATH
deno: SKIPPED: Cannot find "deno" in PATH
composer: SKIPPED: Cannot find "composer" in PATH
krew: SKIPPED: Cannot find "kubectl-krew" in PATH
gem: SKIPPED: Cannot find "gem" in PATH
sheldon: SKIPPED: Cannot find "sheldon" in PATH
rtcl: SKIPPED: Cannot find "rupdate" in PATH
gcloud: SKIPPED: Cannot find "gcloud" in PATH
micro: OK
raco: SKIPPED: Cannot find "raco" in PATH
Flatpak: OK
snap: SKIPPED: Cannot find "snap" in PATH
pihole: SKIPPED: Cannot find "pihole" in PATH
Firmware upgrades: OK
Restarts: OK
 2021-04-11T21:37:24.605Z DEBUG topgrade::terminal         > Desktop notification: Topgrade finished sucessfully
</pre></details>
